app/views/chat_channels/index.html.erb

```diff
@@ -14,7 +14,8 @@
       data-pusher-key="<%= ApplicationConfig["PUSHER_KEY"] %>"
       data-github-token="<%= @github_token %>"
       data-chat-channels="<%= [] %>"
-      data-chat-options="<%= { showChannelsList: true, showTimestamp: true, activeChannelId: @active_channel&.id, currentUserId: current_user.id }.to_json %>">
+      data-chat-options="<%= { showChannelsList: true, showTimestamp: true, activeChannelId: @active_channel&.id, currentUserId: current_user.id }.to_json %>"
+      data-tag-moderator=<%= { isTagModerator: current_user.tag_moderator? }.to_json %>>
```

Comment:
refactor (blocking): This needs to be in double quotes as it renders as an HTML attribute with a value. As @citizen428 [mentioned](https://github.com/forem/forem/pull/9741/#issuecomment-733511751), this is the culprit.

```suggestion
      data-tag-moderator="<%= { isTagModerator: current_user.tag_moderator? }.to_json %>">
```